### ISSUE TITLE
Highlight exact fuzzy matches with different color

### ISSUE THREAD

**Issue by ecosse3** [nvim-telescope/telescope.nvim#1999]
**Is your feature request related to a problem? Please describe.**

Hi. I think it would be nice to have exact matches highlighted with different color. It's a bit hard to difference exact matches when there are a lot of results in the telescope window.

I think it would be much faster to find the results we are looking for in any picker (live grep, find files, git commits etc.)

![image](https://user-images.githubusercontent.com/17691791/173184068-c0805023-7b71-4e32-8079-c78bf0d4d32f.png)

Currently, telescope is using `TelescopeMatching` highlight group for that.

**Describe the solution you'd like**
Highlight exact matches with different highlight group.

**Describe alternatives you've considered**
None

**Comment by Conni2461** [comment]
Sounds cool, might be harder to do and will cost performance. We can look into it after #1491 

Scheduled for post 0.1 release
